src/Adobe.ACPAnalytics.tvOS/Adobe.ACPAnalytics.tvOS.csproj

```diff
@@ -60,7 +60,7 @@
   <ItemGroup>
      <PackageReference Include="Adobe.ACPCore.tvOS" Version="1.0.1" />
      <PackageReference Include="NuGet.Build.Packaging">
-       <Version>0.2.3</Version>
+       <Version>0.2.2</Version>

```

Comment:
why we downgraded here?

Reply:
It fixes a build failure for the tvos target, latest version of [nuget.build.packaging is 0.2.2 ](https://www.nuget.org/packages/NuGet.Build.Packaging/).